src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs

```diff
@@ -17,38 +17,89 @@ namespace Microsoft.ML.Trainers.Online
 {
     public abstract class AveragedLinearArguments : OnlineLinearArguments
     {
+        /// <summary>
+        /// <a href="tmpurl_lr">Learning rate</a>.
```

Comment:
>Learning rate [](start = 32, length = 13)

Remarks?

Reply:
there is no remarks for members. It is <value>. IMO summary will do too. 

---
In reply to: [256223629](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256223629) [](ancestors = 256223629)

Reply:
IMO, if it is short, it can stay in summary. This does need more text, if that's what you mean by `remarks`. 

---
In reply to: [256223629](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256223629) [](ancestors = 256223629)

Reply:
i was thinking the same as Senja. The remark will be simply the link, so we might as well leave it in the summary.

---
In reply to: [256251344](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256251344) [](ancestors = 256251344,256223629)